libtool: link: g++ -march=i686 -mtune=generic -O2 -pipe -fno-strict-aliasing -DNDEBUG -I/usr/include -I/usr/include -DBTG_DEBUG=0 -DBTG_OPTION_SAVESESSIONS=1 -DBTG_OPTION_EVENTCALLBACK=1 -DBTG_OPTION_UPNP=1 -DBTG_OPTION_USECYBERLINK=0 -DBTG_REV=TRUNK -DBTG_INTERNAL_XMLRPC=1 -march=i686 -mtune=generic -O2 -pipe -fno-strict-aliasing -Wl,--hash-style=both -o .libs/btgpasswd btgpasswd-arg.o btgpasswd-btgpasswd.o btgpasswd-dconfig.o  -L/usr/lib ../../bcore/.libs/libbtgcore.so /usr/lib/libtorrent-rasterbar.so -lssl -lcrypto -lboost_iostreams-mt /usr/lib/libgnutls.so -lz /usr/lib/libtasn1.so /usr/lib/libgcrypt.so /usr/lib/libgpg-error.so -lboost_program_options-mt -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt ./.libs/libauth.a ../../external/ini/.libs/libini.a /usr/lib/libstdc++.so -lm -pthread -Wl,-rpath -Wl,/usr/lib/btg
btgpasswd-btgpasswd.o: In function `std::basic_string<char, std::char_traits<char>, std::allocator<char> > boost::posix_time::to_simple_string_type<char>(boost::posix_time::ptime)':
btgpasswd.cpp:(.text._ZN5boost10posix_time21to_simple_string_typeIcEESbIT_St11char_traitsIS2_ESaIS2_EENS0_5ptimeE[std::basic_string<char, std::char_traits<char>, std::allocator<char> > boost::posix_time::to_simple_string_type<char>(boost::posix_time::ptime)]+0x39c): undefined reference to `boost::gregorian::greg_month::as_short_string() const'
collect2: ld returned 1 exit status
make[3]: *** [btgpasswd] Error 1
make[3]: Leaving directory `/var/tmp/fst/src/btg-0.9.8/daemon/auth'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/var/tmp/fst/src/btg-0.9.8/daemon'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/var/tmp/fst/src/btg-0.9.8'
make: *** [all] Error 2

This should fix the above error.

diff --git a/bcore/Makefile.am b/bcore/Makefile.am
index 17afeee..a3c6717 100644
--- a/bcore/Makefile.am
+++ b/bcore/Makefile.am
@@ -131,7 +131,8 @@ urlstatus.cpp
 
 libbtgcore_la_LIBADD = @XMLRPC_LDFLAGS@ @LIBTORRENT_LIBS@ @BOOST_IOSTREAMS_LIB@ \
 @LIBGNUTLS_LIBS@ @BOOST_PROGRAM_OPTIONS_LIB@ \
-@BOOST_THREAD_LIB@ @BOOST_FILESYSTEM_LIB@ @BOOST_SYSTEM_LIB@
+@BOOST_THREAD_LIB@ @BOOST_FILESYSTEM_LIB@ @BOOST_SYSTEM_LIB@ \
+@BOOST_DATE_TIME_LIB@
 
 if BUILDINTERNALXMLRPC
   libbtgcore_la_LIBADD += $(top_builddir)/external/xmlrpc-epi/src/libxmlrpc.la
